lib/player.rb

```diff
@@ -0,0 +1,12 @@
+class Player
+  attr_reader :name
+  attr_accessor :sign
```

Comment:
Only use `attr_accessors` if you really need it. Please be aware that by using it you allow writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).